### PR TITLE
Updating and restyling buttons on unit overview page

### PR DESCRIPTION
**Pull request by hannahbergam**
This reworks the styling of the unit overview page and the included buttons. I couldn't do one button without fixing the others or the page would look wonky! This also makes all these buttons tab navigable:)

I also ended up editing the margins of that section (for both surveys and not) to fix the top and bottom spacing- first and last buttons were not evenly spaced.

See before and after photos for details:)

Before:
Screen Shot 2022-12-15 at 12 54 20 PM

After:
Screen Shot 2022-12-15 at 1 37 41 PM

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/A11Y-103
- https://codedotorg.atlassian.net/browse/A11Y-215
- https://codedotorg.atlassian.net/browse/A11Y-102
- https://codedotorg.atlassian.net/browse/A11Y-100
- https://codedotorg.atlassian.net/browse/A11Y-152

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/progress/LessonLock.jsx [hunk 1]
```diff
@@ -52,7 +51,8 @@ const styles = {
   button: {
     paddingLeft: 0,
     paddingRight: 0,
-    width: '100%'
+    width: '100%',
+    margin: '5px 0px 0px 0px'
```
Comment (hannahbergam) [inline apps/src/templates/progress/LessonLock.jsx:56]: I list it this way so it overrides a button margin styling that uses 5 px everywhere

File: apps/src/templates/progress/ProgressLessonTeacherInfo.jsx [hunk 2]
```diff
@@ -177,6 +174,7 @@ const styles = {
   },
   button: {
     width: '100%',
+    margin: '5px 0px 0px 0px',
```
Comment (hannahbergam) [inline apps/src/templates/progress/ProgressLessonTeacherInfo.jsx:178]: same as above^ I write it this way to override a margin button styling

File: apps/src/templates/progress/LessonLock.jsx (hunk 1 above)
Comment (molly-moen) [reply apps/src/templates/progress/LessonLock.jsx:56]: Can you add a comment to the code so we don't accidentally change it and stop overriding that style?

**Review by molly-moen — APPROVED** [review]
LGTM, just one note on commenting :)

**Comment by hannahbergam** [comment]
> LGTM, just one note on commenting :)

Good idea! Added comments:)

**Pull request merged**